bundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Netgen\Bundle\EzPlatformSiteApiBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\Configuration as SiteAccessConfiguration;
+
+class Configuration extends SiteAccessConfiguration
+{
+    protected $rootNodeName;
+
+    public function __construct($rootNodeName)
+    {
+        $this->rootNodeName = $rootNodeName;
+    }
+
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root($this->rootNodeName);
+
+        $this->addConfiguration($rootNode);
+
+        return $treeBuilder;
+    }
+
+    protected function addConfiguration($rootNode)
+    {
+        $systemNode = $this->generateScopeBaseNode($rootNode);
+        $systemNode
+            ->booleanNode('override_url_alias_view_action')
```

Comment:
The thing is there is no such concept in the code, this actually **is** the action when the view is resolved through URL alias.